Aztec/Classes/GUI/FormatBar/FormatBar.swift

```diff
@@ -716,10 +794,18 @@ extension FormatBar: UIScrollViewDelegate {
     }
 }
 
+// MARK: - Constants
+extension FormatBar {
+    struct Constants {
```

Comment:
Just a single comment because I've seen we're aiming towards this in several PRs.

Any chance we can opt for more meaningful / descriptive names for more atomic grouping?  And maybe even drop the word "Constants" entirely from it?

Having it named `Constants` will mean it'll end up holding a lot of unrelated values, while we could be using the naming to group them logically.

As an example, instead of:

```swift
Constants.defaultBarHeight
```

Make it:

```swift
Dimensions.defaultHeight
```

Reply:
I see – that makes sense (some of these are already split out into `Animations`). Do you want me to do that in this PR?

Reply:
I don't want to step into the review and force a change, so feel free to move ahead.

I just happened to see the `Constants` grouping in a few PRs and wanted to chime in since I feel like this can be a missed opportunity for us.